homeassistant/components/nam/strings.json

```diff
@@ -39,15 +39,125 @@
     }
   },
   "entity": {
+    "button": {
+      "restart": {
+        "name": "[%key:component::button::entity_component::restart::name%]"
+      }
+    },
     "sensor": {
-      "caqi_level": {
+      "bme280_humidity": {
+        "name": "BME280 [%key:component::sensor::entity_component::humidity::name%]"
```

Comment:
Needs confirmation: is it allowed to mix key with text?

Reply:
No, that is not allowed.

Reply:
Thanks, I'll change this.